Packs/AzureStorageContainer/Integrations/AzureStorageContainer/AzureStorageContainer.py

```diff
@@ -881,21 +1043,18 @@ def main() -> None:
     args: Dict[str, Any] = demisto.args()
     verify_certificate: bool = not params.get('insecure', False)
     proxy = params.get('proxy', False)
-
     global account_sas_token
     global storage_account_name
     account_sas_token = params['credentials']['password']
     storage_account_name = params['credentials']['identifier']
-    # supported api versions can be found here:
-    # https://learn.microsoft.com/en-us/rest/api/storageservices/previous-azure-storage-service-versions
```

Comment:
Why did you removed it ? if it's no longer valid please add a valid url

Reply:
by mistake, it is now added again .